_posts/2021-04-19-Networked Dystopia.md

```diff
@@ -0,0 +1,78 @@
+
+<!--more-->
+
+### The Data
+I used [IMBD's list of feature films labeled with the keyword "dystopian,"](https://www.imdb.com/search/keyword/?keywords=dystopia&pf_rd_m=A2FGELUUNOQJNL&pf_rd_p=bdc91cb7-0144-4906-b072-b45760c8aa67&pf_rd_r=87SV6VT6763RFZYAG8HZ&pf_rd_s=right-1&pf_rd_t=15051&pf_rd_i=genre&ref_=kw_nxt&mode=detail&page=1&title_type=movie&sort=moviemeter,asc) and analyzed the top 100 movies in the list sorted by popularity. While going through this list, I eliminated movies that did not fully embody the themes of dystopia, including fantasy movies and a documentary, as well as those whose locations were not easily classifiiable, for a total of 94 movies. Since all data for this project was based on a selection of only 94 movies, there may be trends that are not well represented by this data. 
+
+### Labeling the Data
+I used plot summaries found on IMDB, Wikipedia, and fan websites to classify each movie by its setting. Multiple settings were applied to one movie if more than one setting was highly relevant to the plot. This was primarily applied for movies that took place both on Earth and in outer space. In most cases, only the movie's primary setting was used.
+
+Through this process, I began to realize the surprising complexity of my question. Some movies, especially those that took place in space alone, were relatively straightforward to classify. Others, however, became much harder, as I found that two movies set in the same physical location might have entirely different *settings* when considering the impact of the movie's plot and time period. This led me to create multiple classification categories that separated movies between those describing an alternate past, the present, and the future. I chose to distinguish the future separately from the present as a distant future often exhibits different visual and societal features than the audience's more familiar modern setting, even if the name of the setting has not changed. I also chose to separate the setting classifications into global, national, and city-specific levels in order to see if particular locations were more commonly used as dystopian settings. I only included edges at the most specific levels, such that I only included an edge between the film and its city, and not the city's country, in cases where a specific city was identifiable as a setting.
+
+[The spreadsheet of movies and their settings can be found here.](https://docs.google.com/spreadsheets/d/1aJKW8D20lYVDX--59RklBFRlqNi0tgz_dzQ4i5_sINY/edit?usp=sharing)
+
+#### Below is the dictionary containing the labels for the settings and a description of what criteria were used to assign that setting to a given movie:
```

Comment:
I really appreciate you adding this data dictionary--I can see that your work in Introduction to Digital Humanities is definitely informing this particular assignment. This is a really great model for other students or even readers of the blog (and believe it or not, we do have readers) to see the kinds of analytical thinking and organizational principles that go into devising a standard set of criteria with which to analyze a data set.